src/Paket.Core/NugetConvert.fs

```diff
@@ -184,7 +184,8 @@ module NugetEnv =
                   Packages = [for node in doc.SelectNodes("//package") ->
                                     { Id = node.Attributes.["id"].Value
                                       Version = node.Attributes.["version"].Value |> SemVer.Parse
-                                      TargetFramework = node |> getAttribute "targetFramework" } ]}
+                                      TargetFramework = 
+                                        node |> getAttribute "targetFramework" |> Option.map (fun t -> ">= " + t) } ]}
```

Comment:
so this is actually the only change necessary? i.e. we don't need to call the `optimizeRestrictions` function?